spec/controllers/npq_separation/migration/parity_checks_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+RSpec.describe NpqSeparation::Migration::ParityChecksController, type: :request do
```

Comment:
should this be under spec/requests instead?

Reply:
I put it here to be consistent with the migration request spec (which I think was introduced before we started using `spec/requests`), but we did agree to put them there going forward so I'll update 👍 I'll consolidate all the `npq_separation` specs from `/controllers` with `/requests` while I'm at it as there's a few